app/assets/javascripts/bastion/components/views/bst-edit-save-cancel.html

```diff
@@ -1,6 +1,7 @@
 <div ng-show="workingMode">
   <i class="fa fa-spinner inline-icon fa-spin"></i>
-  <span translate>Working</span>
+  <span ng-if='forcedWorkingText' translate>{{ forcedWorkingText }}</span>
```

Comment:
I'm still not sure that this would come through as the actual value of `forcedWorkingText` rather than the string `"{{forcedWorkingText}}"`

Reply:
Hmm..so can we pass the translated string [here](https://github.com/Katello/katello/pull/7997/files#diff-d4e32eb20afc296d5b7c5492fb1faff2R48) from katello instead and remove the translate filter on {{forcedWorkingText}} here? I don't know how this would be evaluated otherwise in bastion?

Reply:
Yes, that sounds like a good plan to me.

Reply:
Updated!